File: back/src/main/java/org/example/hackatonassurance/entities/Accelerometre.java
package org.example.hackatonassurance.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Getter
@Setter
@AllArgsConstructor
@ToString
public class Accelerometre {
    private Double x;
    private Double y;
    private Double z;
    private LocalDate date;
}

File: front/hackaton_project/src/app/shared/model/accelerometre.ts
export class Accelerometre {
  x : number;
  y : number;
  z : number;
  date : string;

  constructor(x: number, y: number, z: number, date: string) {
    this.x = x;
    this.y = y;
    this.z = z;
    this.date = date;
  }
}
